From a40a06a4e167427c4f8925ec01d80c987a02d3b0 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Tue, 13 Nov 2012 14:15:00 +0100
Subject: [PATCH] net: xilinx_emacps: Use dev_err with proper device link

Warning log:
drivers/net/ethernet/xilinx/xilinx_emacps.c: In function 'xemacps_mii_init':
drivers/net/ethernet/xilinx/xilinx_emacps.c:934:1: warning:
label 'err_out_unregister_bus' defined but not used [-Wunused-label]

This patch should be the part of this commit:
net: xilinx_emacps: Use dev_ prefix over pr_
(sha1: 6e9d94cc5d3245616f4b4155590fd7140baa23b1)

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
---
 drivers/net/ethernet/xilinx/xilinx_emacps.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/drivers/net/ethernet/xilinx/xilinx_emacps.c b/drivers/net/ethernet/xilinx/xilinx_emacps.c
index 9c16872..ddf2fc8 100644
--- a/drivers/net/ethernet/xilinx/xilinx_emacps.c
+++ b/drivers/net/ethernet/xilinx/xilinx_emacps.c
@@ -2135,7 +2135,8 @@ static int xemacps_open(struct net_device *ndev)
 
 	rc = pm_runtime_get(&lp->pdev->dev);
 	if (rc < 0) {
-		dev_err("%s pm_runtime_get() failed, rc %d\n", ndev->name, rc);
+		dev_err(&lp->pdev->dev,
+			"%s pm_runtime_get() failed, rc %d\n", ndev->name, rc);
 		goto err_free_rings;
 	}
 
-- 
1.7.0.4

